lib/util.js

```diff
@@ -16,10 +17,10 @@ module.exports = {
         !callback && _.isFunction(options) && (callback = options, options = {});
         return (/^https?:\/\/.*/).test(location) ?
             // Load from URL
-            request.get({ url: location, json: true }, (err, response, body) => {
-                if (err) {
-                    return callback(err);
-                }
+            request.get({
+                url: location, json: true, headers: { 'User-Agent': 'Newman/' + version }
+            }, (err, response, body) => {
+                if (err) { return callback(err); }
```

Comment:
What happened to the function end brace indentation here? Should it not be in new line?